roles/swift-common/tasks/main.yml

```diff
@@ -76,6 +76,16 @@
     - logging
   when: logging.enabled|default('True')|bool
 
+- name: Change cinder swift dir acl
+  acl: name=/var/log/swift state=present default=yes etype={{ item.etype }} permissions={{ item.permission }}
+  with_items:
+    - etype: user
+      permission: rw
+    - etype: group
+      permission: r
+    - etype: other
+      permission: r
```

Comment:
This task should be done in the logging task file, so that it can be picked up with the `logging` tag.